willy/Week6/cv-fullstack/frontend/src/app/cv/cv.component.html

```diff
@@ -0,0 +1,37 @@
+<main class="main__container">
+    <div class="container__izq">
+        <div class="izq__data" *ngFor="let data of personalData">
+            
+            <img class="data__img" src="{{data.img}}" alt="Stan Lee">
+            <p>Name: {{data.name}}</p>
```

Comment:
para hacer las siguientes <p> podrias haber hecho un for, y que creara un tag <p> por cada propiedad del objeto